server/journeys/referral/area-of-work/schemas.ts

```diff
@@ -0,0 +1,20 @@
+import z from 'zod'
+import { Request } from 'express'
+import CsipApiService from '../../../services/csipApi/csipApiService'
+
+export const schemaFactory =
+  (csipApiService: CsipApiService) =>
+  async (req: Request): Promise<z.ZodTypeAny> => {
+    const areaRefDataMap = new Map(
+      (await csipApiService.getReferenceData(req, 'area-of-work')).map(itm => [itm.code, itm]),
+    )
+
+    return z.object({
+      areaOfWork: z
+        .string({ message: 'Select your area of work' })
+        .min(1, 'Select your area of work')
+        .max(12, 'Select your area of work')
```

Comment:
I don't think the min max restrictions are needed any more as the refine bit below should cover it?